dotCMS/src/main/java/com/dotcms/publishing/BundlerUtil.java

```diff
@@ -213,25 +216,33 @@ public static void objectToJSON( Object obj, File f, boolean removeFirst ) {
     public static Object xmlToObject(File f){
     	XStream xstream = new XStream(new DomDriver("UTF-8"));
 
-    	BufferedInputStream input = null;
-		try {
-			input = new BufferedInputStream(Files.newInputStream(f.toPath()));
-			Object ret = xstream.fromXML(input);
-			return ret;
-		} catch (IOException e) {
-			Logger.error(BundlerUtil.class,e.getMessage(),e);
-			return null;
-		}finally{
-			try {
-				input.close();
-			}
-			catch(Exception e){
-
-			}
+        try (InputStream input = new BufferedInputStream(Files.newInputStream(f.toPath()))) {
+            return xstream.fromXML(input);
+		} catch (Exception e) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid catching generic exceptions such as NullPointerException, RuntimeException, Exception in try-catch block](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5632525)